<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>datawindow</class>
 <widget class="QMainWindow" name="datawindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_2" native="true">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>310</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot_3" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>221</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>866</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
